tasks/cv/en/cv-hints.md

```diff
@@ -95,9 +95,8 @@ The following tips are intended to help you complete the "Markdown & Git" and "H
 
 - **Do not press "Merge"!**
 
-16. Go to "RS App," then the "Auto-Test" tab. Choose the "Markdown & Git" task and click the "Submit" button.
+16. Go to "RS App," then the "Auto-Test" tab. At the time of reading this article, the interface has changed so when you go to auto-tests, you select a specific task, in this case CV#1 and just click "start task" and then "submit" .The empty window will be filled with your result, the school bot itself will find your repository by the name you specified during registration and check it out.
```

Comment:
I am not sure it's necessary to specify that the interface has changed. For users it doesn't really matter. I think it's better just to give instructions:

Go to "RS App," then the "Auto-Test" tab, select CV#1, click "start task", click "submit".

Short instructions are the way to go. And it would be also great if you provide a picture. There was a picture on the line 100, if the picture is outdated, it would be nice if you took a new one. Better to show once than explain 5 times.